Tests/js/views/ez-rawcontentview.html

```diff
@@ -19,6 +20,32 @@
 
 <script type="text/x-handlebars-template" id="fieldview-ez-template">fieldview</script>
 
+<script type="text/x-handlebars-template" id="languageswitcherview-ez-template">
+    <div class="ez-selected-language">
+        {{#if otherTranslations}}
+            <a href="#" class="ez-dropdown-list-indicator" data-test="{{ location.id }}">
+                <span class="ez-language-flag flag-icon"></span>
```

Comment:
actually in this test I think the template for the language switch can even be an empty string